### PR TITLE
Fix: Link metadata tree component action with ddl editor

### PR DESCRIPTION
**Pull request by indraraj**
- Jira Issue https://issues.redhat.com/browse/ENTESB-12813

- Insert the column or table name in DDL editor at last selected cursor position.

- Show warning msg if user does not select cursor position before inserting text

- screenshots
image

### CODE REVIEW

File: app/ui-react/syndesis/src/modules/data/locales/data-translations.en.json [hunk 1]
```diff
@@ -77,7 +77,9 @@
   "importVirtualizationTip": "Import a data virtualization",
   "importVirtualizationZipInvalid": "Compressed file is invalid (dv.json not found).",
   "Invalid": "Invalid",
-  "metadataTree": "Metadata Tree",
+  "metadataTree": "Schemas",
```
Comment (mdrillin) [inline app/ui-react/syndesis/src/modules/data/locales/data-translations.en.json:80]: you can remove 'metadataTree' here if not used anymore.

File: app/ui-react/syndesis/src/modules/data/locales/data-translations.it.json [hunk 2]
```diff
@@ -78,6 +78,8 @@
   "importVirtualizationZipInvalid": "Il file compresso non è valido (dv.json non trovato).",
   "Invalid": "Invalido",
   "metadataTree": "Albero dei metadati",
```
Comment (mdrillin) [inline app/ui-react/syndesis/src/modules/data/locales/data-translations.it.json:80]: remove 'metadataTree' if unused

File: app/ui-react/syndesis/src/modules/data/locales/data-translations.ru.json [hunk 3]
```diff
@@ -78,6 +78,8 @@
   "importVirtualizationZipInvalid": "Сжатый файл недействителен (dv.json не найден).",
   "Invalid": "Некорректный",
   "metadataTree": "Дерево метаданных",
```
Comment (mdrillin) [inline app/ui-react/syndesis/src/modules/data/locales/data-translations.ru.json:80]: remove 'metadataTree' if unused

**Comment by indraraj** [comment]
Thanks, Mark. I have removed the "metadataTree" from 18nfiles in next pr here https://github.com/syndesisio/syndesis/pull/8455

**Comment by mdrillin** [comment]
Looks good, thanks!

**Pull request merged**